### DISCUSSION TITLE
Feature Request: Remove Time & Date + Remove Notifications on System Tray

### DISCUSSION THREAD

**Discussion by segregates** [valinet/ExplorerPatcher#2893] [Ideas]
Disabling the time & date is an option in the windows 11 menu, but for some reason it doesn't seem to work with Explorer Patcher, would it be possible to add a feature to remove the time & date in the system tray, as well as remove the notifications icon?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Reply by milestprower92** [reply]
this is not an issue

**Reply by pyrates999** [reply]
If you're not willing to provide the information, then I can't help you.

**Comment by milestprower92** [comment]
> Disabling the time & date is an option in the windows 11 menu, but for some reason it doesn't seem to work with Explorer Patcher, would it be possible to add a feature to remove the time & date in the system tray, as well as remove the notifications icon?

![imagen](https://github.com/valinet/ExplorerPatcher/assets/131640765/fbfca583-23e4-484f-94c7-c72cbe497e05)

click this and disable the icons you don't want

**Reply by pixelloop** [reply]
I second this. 

I record videos for clients and don't want them to see when exactly I am doing this. 

My workaround is using the Win11ClockToggler app from https://github.com/jmalarcon/Win11ClockToggler

It says it's no longer needed (as Win 11 added the option) but ExplorerPatcher doesn't obey this.

**Reply by pyrates999** [reply]
did you change your taskbar to windows 10 or windows 11 taskbar?

**Reply by pixelloop** [reply]
Windows 10.

**Reply by pyrates999** [reply]
Try this: https://www.thewindowsclub.com/hide-clock-and-date-from-taskbar-windows-10

Use the registry method.

**Reply by pixelloop** [reply]
I just use the Windows Clock Toggle App, but it would be nice if this worked on its own. I don't want to edit registry because I want to turn the clock on and off with one or two clicks. The Clock Toggler does that. 

I appreciate your input though.

**Reply by pyrates999** [reply]
Ok, then please wait for a developer to respond.
